src/overlays/actors/ovl_En_Mag/z_en_mag.c

```diff
@@ -319,7 +319,7 @@ void EnMag_DrawImageRGBA32(Gfx** gfxp, s16 centerX, s16 centerY, u8* source, u32
     textureSize = (width * textureHeight) << 2;
     textureCount = remainingSize / textureSize;
     if ((remainingSize % textureSize) != 0) {
-        textureCount += 1;
+        textureCount++;
```

Comment:
not a fan, because this is a non-repeating operation and the 1 is meaningful: it's one more fragment

same in `z_kaleido_equipment.c`